rules/cloudflare_rules/cloudflare_httpreq_bot_high_volume.yml

```diff
@@ -7,7 +7,6 @@ LogTypes:
   - Cloudflare.HttpRequest
 Tags:
   - Cloudflare
```

Comment:
Action needed, I think: 

While this detection doesn't require configuration, there is a logical bug in it that a `BotScore` of 0 means "not computed". We should be returning False when BotScore is zero or  greater than 30 https://developers.cloudflare.com/bots/concepts/bot-score/#bot-groupings

Additionally, I think we should add a conditional for the action being block.

Reply:
Flipping this to false for now while we update this detection in another branch.